gffquant/profilers/feature_quantifier.py

```
@@ -127,50 +149,83 @@ def process_counters(
         )
 
-        if not external_gene_counts:
-            count_writer.write_gene_counts(
-                count_annotator.gene_counts,
-                count_annotator.scaling_factors["total_gene_uniq"],
-                count_annotator.scaling_factors["total_gene_ambi"]
+        logger.info("FC-index: %s", str(list(functional_counts.index.keys())[:10]))
+        logger.info("FC-counts: %s", str(functional_counts.counts[0:10, :]))
+
+        categories = self.adm.get_categories()
+        for category, category_sum in zip(categories, category_sums):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)